include/dxc/Support/HLSLOptions.td

```diff
@@ -284,6 +284,8 @@ def validator_version : Separate<["-", "/"], "validator-version">, Group<hlslcom
   HelpText<"Override validator version for module.  Format: <major.minor> ; Default: DXIL.dll version or current internal version.">;
 def print_after_all : Flag<["-", "/"], "print-after-all">, Group<hlslcomp_Group>, Flags<[CoreOption, HelpHidden]>,
   HelpText<"Print LLVM IR after each pass.">;
+def print_after : Separate<["-", "/"], "print-after">, Group<hlslcomp_Group>, Flags<[CoreOption, HelpHidden]>,
+  HelpText<"Print LLVM IR after a specific pass.">;
```

Comment:
This is nice, but I immediately wonder if this could be made to work when supplied multiple times on the command line, collecting a set of passes you are interested in, instead of being limited to one.

Reply:
Also, a `-print-on-change` (or some better name) where it prints before and after for a matching pass only when it changes the module would be useful (and would require printing before the pass into a stream that's unused if unchanged).  Of course, this is just an idea for potential future use, not really relevant to this particular PR.

Reply:
Moving all this to a different PR now.